src/main/java/seedu/address/model/person/Birthday.java

```diff
@@ -40,12 +40,14 @@ public Birthday(String birthday) {
      * Returns true if a given string is a valid name.
      */
     public static boolean isValidBirthday(String test) {
-        if (test == null || test.isEmpty()) {
+        if (test == null || test.isBlank()) {
             return true;
         }
+        test = test.strip();
```

Comment:
Perhaps can consider using strip() or trim() in parseBirthday from ParserUtil although this works as well